src/mo_frontend/menhir_error_reporting.ml

```diff
@@ -150,26 +150,6 @@ module Make
       ) []
     )
 
-  (* We drive the parser in the usual way, but records the last [InputNeeded]
```

Comment:
I don't quite grok why this code has disappeared, was it unused?

Reply:
See https://github.com/dfinity/motoko/pull/4959/commits/95e29ec137cfef6a1b8e7b1315f84d17dda97893
I have inlined it into `parsing.ml`